PROJECT

```diff
@@ -8,18 +8,18 @@ resources:
 - api:
     crdVersion: v1
     namespaced: false
+  controller: true
   domain: kubewarden.io
   group: policies
-  kind: ClusterAdmissionPolicy
-  path: github.com/kubewarden/kubewarden-controller/apis/policies/v1alpha2
-  version: v1alpha2
+  kind: PolicyServer
+  path: github.com/kubewarden/kubewarden-controller/apis/policies/v1alpha3
+  version: v1alpha3
```

Comment:
I know we discussed that yesterday, but do we have to update the api version? Given the new field is optional, can't we just add it to the existing v1alpha2 policy?

Moreover, if we are forced to add a new version of the object, the old one must stay around for some time for compatibility reasons.